sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -105,26 +105,29 @@ public virtual async Task<TrainingOperation> StartTrainingAsync(Uri trainingFile
         }
 
         /// <summary>
+        /// Get detailed information about a custom model.
         /// </summary>
-        /// <param name="modelId"></param>
-        /// <param name="cancellationToken"></param>
+        /// <param name="modelId">The ID of the model to retrieve.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns></returns>
         [ForwardsClientCalls]
         public virtual Response<CustomFormModel> GetCustomModel(string modelId, CancellationToken cancellationToken = default)
         {
-            throw new NotImplementedException();
+            Response<Model_internal> response = ServiceClient.GetCustomModel(new Guid(modelId), true, cancellationToken);
```

Comment:
small request: for the second parameter - could we add the parameter name so it's easy to see what the true signifies?

Reply:
updated